### ISSUE TITLE
Its not supporting with Angular16

### ISSUE THREAD

**Issue by sadhana98** [kzrfaisal/angular-file-uploader#124]

**Comment by sadhana98** [comment]
When I was trying to install angular file uploader, I faced an issue which says found @angular/common 16.1.0 in root try with legacy --legacy-peer-deps then finally I installed.
But even after installing I am facing below issue
`**This likely means that the library (angular-file-uploader) which declares AngularFileUploaderModule is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.**`

**Comment by volpettone5000** [comment]
Yes, here same trouble... I've just upload my project to Angular 16

**Comment by kzrfaisal** [comment]
We're soon going to release an update which will support angular 16.

**Comment by melaniesavas** [comment]
How soon will this be? We are about to upgrade to Angular 17.

**Comment by volpettone5000** [comment]
At the end I've used the html file uploader and it works fine...

**Comment by elgibor-solution** [comment]
is this package already depreacted?
